engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -65,15 +98,38 @@ public void resetCamera() {
     }
 
```

Comment:
Something I simply don't know rather than a problem with the code: once a command is registered one has to bind it to a string or a key to use it respectively in a console or via keyboard and that is done elsewhere. Is that how it works?